addons/storyshots/package.json

```diff
@@ -45,6 +45,7 @@
     "@storybook/addons": "^3.3.3",
     "babel-core": "^6.26.0 || ^7.0.0-0",
     "react": "*",
-    "react-test-renderer": "*"
+    "react-test-renderer": "*",
+    "vue": "^2.5.13"
```

Comment:
I think we should remove the framework peer dependencies instead of adding them

Reply:
Yeah, I think so too.